packages/arb-token-bridge-ui/src/components/common/SiteBanner.tsx

```diff
@@ -1,39 +1,58 @@
-import dayjs from 'dayjs'
 import { useEffect, useState } from 'react'
+import dayjs from 'dayjs'
+import utc from 'dayjs/plugin/utc'
 import { ExternalLink } from './ExternalLink'
 import { ArbitrumStatusResponse } from '../../pages/api/status'
 import { getAPIBaseUrl } from '../../util'
 
-const SiteBannerArbiscanIncident = () => {
+dayjs.extend(utc)
+
+const SiteBannerArbiscanIncident = ({
+  type
+}: {
+  type: 'arbitrum-one' | 'arbitrum-nova'
+}) => {
+  const chainName = type === 'arbitrum-one' ? 'Arbitrum One' : 'Arbitrum Nova'
```

Comment:
nit: this could be
```
const isArbitrumOne = type === 'arbitrum-one'
```
and reused for all

Reply:
Updated